### ISSUE TITLE
UITableViewController last button under the visible part of activity

### ISSUE THREAD

**Issue by zemadara** [beryu/CustomizableActionSheet#10]
Hello, 
when i create a CustomizableActionSheet with buttons, in a UITableViewController, the last button is below in the UITableViewController but not visible, is it possible to display it with the others in the visible part of my UITableViewController ?

**Comment by beryu** [comment]
Thanks for reporting, and sorry to late response.
Can you show me example codes?

**Comment by zemadara** [comment]
I have fix this issue it was my code sorry

**Comment by beryu** [comment]
I'm glad to hear you fixed it!
I will close this issue.

**Comment by GustavoAlvarez** [comment]
How did you fix it?, I have the same problem, this is my code:

```
var items = [CustomizableActionSheetItem]()
        // Setup custom view
        if let sampleView = UINib(nibName: "SampleView", bundle: nil).instantiate(withOwner: self, options: nil)[0] as? SampleView {
            sampleView.delegate = self
            sampleView.delegate = self
            let sampleViewItem = CustomizableActionSheetItem(type: .view, height: 200)
            sampleViewItem.view = sampleView
            items.append(sampleViewItem)
            
            // Setup button
            let closeItem = CustomizableActionSheetItem()
            closeItem.type = .button
            closeItem.label = "Close"
            closeItem.selectAction = { (actionSheet: CustomizableActionSheet) -> Void in
                actionSheet.dismiss()
            }
            items.append(closeItem)
            
        }
        
        let actionSheet = CustomizableActionSheet()
        self.actionSheet = actionSheet
        actionSheet.showInView(self.view, items: items)
```

**Comment by beryu** [comment]
@GustavoAlvarez Thank you for reporting. And I reproduced below.

![result](https://user-images.githubusercontent.com/202968/38906251-6c5d3be2-42f0-11e8-8fd2-593fe73a5e9a.png)

It has occured in iOS11 Simulator. 
And in iOS10, I can't reproduced it.

I reopen this issue and examine about it.
But maybe it'll be late because I'm working hard in weekdays.
Thank you for waiting 🙇

**Comment by GustavoAlvarez** [comment]
I kinda solve it using a UIViewController instead of a TableViewController and embed the table view, then adding an observer when i click on a cell to display the ActionSheet on my UIViewController.
I think this is not the best solution but it works for now
-Sorry for my english

**Comment by beryu** [comment]
@GustavoAlvarez I fixed this in #28 .
And released as 1.2.1 now!
https://github.com/beryu/CustomizableActionSheet/releases/tag/1.2.1

| 1.2.0 | 1.2.1 |
| --- | --- |
| ![1.2.0](https://user-images.githubusercontent.com/202968/39079008-aff78650-454d-11e8-94a7-1f944777e594.png) | ![1.2.1](https://user-images.githubusercontent.com/202968/39079010-b7ab2794-454d-11e8-8f19-6676d75a0217.png) |
